tests/testsuite/metadata.rs

```diff
@@ -192,7 +192,9 @@ optional_feat = []
                 ],
                 "features": {
                   "default": [
-                    "default_feat"
+                    {
+                       "Feature": "default_feat"
+                    }
                   ],
```

Comment:
I only just notest that this change, after the merge. I don't know why/if this change in structures is ok. Will this break things using `metadata`?

Reply:
Yeah, this definitely will break consumers that look at the `features` field. Could we preserve the old format here?

Reply:
Oops -- sorry, I thought that was only used for testing. Will follow up with a fix.